examples/TestDriver/src/topNavigator.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { createBottomTabNavigator, createAppContainer } from 'react-navigation';
+
+import TouchablesPage from './pages/touchables';
+
+import Heap from '@heap/react-native-heap';
+
+Heap.setAppId('2084764307');
```

Comment:
Why are we setting this statically, vs in a `componentDidMount` or something?

I don't doubt there's a good reason but can you walk me through it?

Reply:
No particular reason, other than adding a React Navigation "wrapper" on things means there isn't really a convenient `componentDidMount` to put it in.  This will essentially have the same effect, and should only be run once (but is also idempotent, so doesn't really matter.)